stripe-patrons-data/build.sbt

```diff
@@ -10,6 +10,7 @@ scalacOptions ++= Seq(
 
 libraryDependencies ++= Seq(
   "software.amazon.awssdk" % "dynamodb" % awsClientVersion2,
+  "software.amazon.awssdk" % "ssm" % awsClientVersion2,
   "com.amazonaws" % "aws-java-sdk-ssm" % awsClientVersion,
```

Comment:
Is this supposed to removed?

Reply:
Yes it is! I'm confused, where are you seeing that? The diff shows it as removed for me. Was that looking at a specific commit maybe?
<img width="592" height="213" alt="Screenshot 2025-07-21 at 09 54 44" src="https://github.com/user-attachments/assets/a253a33f-e6de-4f3c-975d-8766c09334b5" />